tests/baselines/reference/bundledNodeDTSFailsWithFlag.js

```diff
@@ -0,0 +1,50 @@
+//// [tests/cases/conformance/declarationEmit/bundledNodeDTSFailsWithFlag.ts] ////

```

Comment:
this is named "FailsWithFlag", but I can't figure out how it fails.

Reply:
Thanks!